appveyor.yml

```diff
@@ -2,21 +2,29 @@
 version: '1.6.{build}'
 image: Visual Studio 2017
 clone_depth: 1
+skip_tags: true
+skip_branch_with_pr: true
 
```

Comment:
Does this address your comment from #33?

Reply:
The opposite, enabling this flag means we have to disable the required status check for branches, because AppVeyor will only build the PR and not the branch if they refer to the same commit.

Reply:
Ah ok. I'll have to look into the docs on how to disable certain things.

Reply:
Just go to the repo settings -> Branches -> Protected Branches -> master -> Required Status Checks -> untick appveyor/branch

If you make me an admin I can also do it